src/suites/cts/validation_test.ts

```diff
@@ -0,0 +1,233 @@
+      this.device.createBindGroup({
+        bindings: [
+          {
+            binding: 0,
+            resource,
+          },
+        ],
+        layout: bindGroupLayout,
+      });
+
+      await this.expectUncapturedError();
+    }
+  }
+
+  setupTestForDynamicOffsets(): { bindGroup: GPUBindGroup; bindGroupLayout: GPUBindGroupLayout } {
+    const MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT = 256;
+
+    const uniformBuffer = this.device.createBuffer({
+      size: 2 * MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT + 8,
+      usage: GPUBufferUsage.UNIFORM,
+    });
+
+    const storageBuffer = this.device.createBuffer({
+      size: 2 * MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT + 8,
+      usage: GPUBufferUsage.STORAGE,
+    });
+
+    const bindGroupLayout = this.device.createBindGroupLayout({
+      bindings: [
+        {
+          binding: 0,
+          visibility: GPUShaderStage.COMPUTE | GPUShaderStage.FRAGMENT,
+          type: 'uniform-buffer',
+          dynamic: true,
+        },
+        {
+          binding: 1,
+          visibility: GPUShaderStage.COMPUTE | GPUShaderStage.FRAGMENT,
+          type: 'storage-buffer',
+          dynamic: true,
+        },
+      ],
+    });
+
+    const BINDING_SIZE = 9;
+
+    const bindGroup = this.device.createBindGroup({
+      bindings: [
+        {
+          binding: 0,
+          resource: {
+            buffer: uniformBuffer,
+            offset: 0,
+            size: BINDING_SIZE,
+          },
+        },
+        {
+          binding: 1,
+          resource: {
+            buffer: storageBuffer,
+            offset: 0,
+            size: BINDING_SIZE,
+          },
+        },
+      ],
+      layout: bindGroupLayout,
+    });
+
+    return { bindGroup, bindGroupLayout };
+  }
+
+  async expectUncapturedErrorInComputePass(dynamicOffsets: number[]): Promise<void> {
+    const { bindGroup, bindGroupLayout } = this.setupTestForDynamicOffsets();
+    const module = this.device.createShaderModule({
+      code: GLSL(
+        'compute',
+        `#version 450
+          const uint kTileSize = 4;
+          const uint kInstances = 11;
+
+          layout(local_size_x = kTileSize, local_size_y = kTileSize, local_size_z = 1) in;
+          layout(std140, set = 0, binding = 0) uniform UniformBuffer {
+              float value1;
+          };
+          layout(std140, set = 0, binding = 1) buffer SBuffer {
+              float value2;
+          } dst;
+          void main() {
+          }
+        `
+      ),
+    });
+    const pipelineLayout = this.device.createPipelineLayout({
+      bindGroupLayouts: [bindGroupLayout],
+    });
+    const pipeline = this.device.createComputePipeline({
+      computeStage: { module, entryPoint: 'main' },
+      layout: pipelineLayout,
+    });
+    const encoder = this.device.createCommandEncoder();
+    const computePass = encoder.beginComputePass();
+    computePass.setPipeline(pipeline);
+
+    //@ts-ignore TODO:Update @webgpu/types
```

Comment:
Should be possible to use `as any` and use `// tslint:disable-next-line: no-any` instead

Reply:
It's not as it is a TypeScript error. See https://github.com/Microsoft/vscode-tslint/issues/368#issuecomment-406175162
Hopefully with updated @webgpu/types, this won't be needed anymore.

Reply:
AFAICT, `as any` should change it from a ts error to a tslint error, then you can suppress the latter. I do this in some other places too (just grep for `no-any`)